packages/wdio-config/src/utils.ts

```diff
@@ -182,7 +182,9 @@ export function makeRelativeToCWD (files: (string | string[])[] = []): (string |
 
         returnFiles.push(file.startsWith('file:///')
             ? url.fileURLToPath(file)
-            : path.resolve(process.cwd(), file))
+            : (file.includes('/'))
```

Comment:
Can you explain this change?

Reply:
Here is what happens if we use only the spec file name `test.spec.js` in cli this method will return the spec file path with CWD but we need only the file name to run the test. and the second case if we give the path of spec `./src/spec/test.spec.js` in cli then we need the spec path with CWD path to find the relevant file.

So I have added a check like if we see any `/` in --spec cli then return `path.resolve(process.cwd(), file))` otherwise return only spec file name.

Reply:
And I have checked this is working as expected if you have any other suggestions then please give me.

Reply:
@christian-bromann  I have checked on Windows and it is working perfectly. And also I have added a comment as per your suggestion please take a look again.

Reply:
This change does not work for me when testing on my Windows machine and it also breaks passing relative paths for me. Tested on Windows 11, Node 18, with both the Windows Terminal and CMD.

Reply:
Hi @erwinheitzman 
Thanks for checking this I have added fix can you please check?